Decred Wallet/Custom Views/FloatingLabelInputs/FloatingPlaceholderLabel.swift

```diff
@@ -0,0 +1,109 @@
+//
+//  FloatingPlaceholderLabel.swift
+//  Decred Wallet
+//
+// Copyright (c) 2019 The Decred developers
+// Use of this source code is governed by an ISC
+// license that can be found in the LICENSE file.
+
+import UIKit
+
+@IBDesignable
+class FloatingPlaceholderLabel: UILabel {
+    @IBInspectable var topPadding: CGFloat = 0
+    @IBInspectable var bottomPadding: CGFloat = 0
+    @IBInspectable var leftPadding: CGFloat = 4
+    @IBInspectable var rightPadding: CGFloat = 4
+
+    var topConstraint: NSLayoutConstraint?
+
+    let normalColor: UIColor = UIColor.appColors.lightBluishGray
+    var activeColor: UIColor = UIColor.appColors.lightBlue
+    let normalFontSize: CGFloat = 16
+    let smallerFontSize: CGFloat = 14
+
+    override var text: String? {
+        didSet {
+            // set position after the text did set, since the label position calculated in real-time using the height of the label
+            self.setPosition()
+        }
+    }
+
+    override init(frame: CGRect) {
+        super.init(frame: frame)
+        self.initView()
+    }
+
+    required init?(coder aDecoder: NSCoder) {
+        super.init(coder: aDecoder)
+        self.initView()
+    }
+
+    public func initView() {
+        self.backgroundColor = UIColor.white
+        self.clipsToBounds = true
+        self.translatesAutoresizingMaskIntoConstraints = false
+        self.textColor = self.normalColor
+        self.setPosition()
+    }
+    
+    private func setPosition() {
+        guard let floatingLabelInput = superview as? FloatingLabelProtocol else { return }
+
+        self.font = self.font.withSize(floatingLabelInput.isParentEmpty() ? self.normalFontSize : self.smallerFontSize)
+        self.sizeToFit()
+        let calculatedTopConstraintValue = floatingLabelInput.isParentEmpty() ? (self.superview!.frame.size.height - self.intrinsicContentSize.height) / 2 : self.intrinsicContentSize.height / -2
```

Comment:
Let's place an empty line **before** this line. The previous 2 lines are not directly related to this.